docs.merkely.com/content/client_reference/merkely_report_env_k8s.md

```diff
@@ -45,7 +45,7 @@ merkely report env k8s prod -k /path/to/kube/config
   -x, --exclude-namespace strings   The comma separated list of namespaces regex patterns NOT to report artifacts info from. Can't be used together with --namespace.
   -h, --help                        help for k8s
   -i, --id string                   The unique identifier of the source infrastructure of the report (e.g. the K8S cluster/namespace name). If not set, it is defaulted to environment name.
-  -k, --kubeconfig string           The kubeconfig path for the target cluster. (default "$HOME/.kube/config")
+  -k, --kubeconfig string           The kubeconfig path for the target cluster. (default "/Users/mikelong/.kube/config")
```

Comment:
since this is auto generated maybe we don't need to care about that, but how did your $HOME ended up here?? 🤔

Reply:
Interesting...I've no idea actually

Reply:
I've reverted this manually, but we should figure out if there is a better way than manually doing this.

Reply:
the default path is generated dynamically when the binary runs (and docs are generated by running a command from the binary). That's fine for the help text because you get a personalized default printed. But we need to make sure it's left as $HOME when we generate docs. We can set a DEV env variable in the make file and don't find the actual home dir when that variable is set to true.